src/node-sim.lisp

```diff
@@ -3,6 +3,8 @@
 (in-package :emotiq/sim)
 
 (defun initialize (&key
```

Comment:
@guitarvydas With so many variables being used in `intitialize` it is probably a good idea to have more of an API for setting and inspecting the current state of how the simulator is set up to run.  But let's see how the coming configuration for real network transport shakes out (i.e. absorbing Shannon's TCP networking configuration mechanism) before extending this.